challenge_7/python/py2k5/findMissingNumber.py

```diff
@@ -0,0 +1,14 @@
+
+def main():
+	numbers = [ 0,1,3,4,2,5,7 ]
+        tot = len(numbers)
+	if 0 not in numbers: 
+		print 0
+	else:
+		for num in numbers:
+			if num +1 < tot and num+1 not in numbers:
```

Comment:
'num + 1 not in numbers is O(n),  and you put it in the for loop, together it's O(n square) complexity?